lib/cyclops/RPC_defs.h

```diff
@@ -1,13 +1,80 @@
 /** @file RPC_defs.h
     @brief Defines a few constants of the RPC Format Specification.
            This file is part of CL.
-  
-    @section header-desc HEADER Field Description
-    ~~~~~~~~~~~~~~~~~~~~
-        <2: channel> <6: command>
-        [7:6] Channel
-        [5:0] Command
-    ~~~~~~~~~~~~~~~~~~~~
+    
+    @page rpc-desc RPC Format Description
+
+    @tableofcontents
+
+    @section rpc-overview Overview
+    There are two kinds of Serial packets, single byte and multi byte.
+    The size of a packet is determined by MSB of header and the ``command`` field.
+
+    @section S-header-desc Single Byte Packets
+    Single byte Headers are distinguished from other headers by the MSB bit. For
+    Single byte headers, this bit is always set
+    
+    Field       | Bits  | Description
+    ----------- | ----- | -----------
+    Reserved    | [7]   | Always ``1``.
+    ``channel`` | [6:3] | Determines if "command" is applied on the Channel @par 1: Command is appplied on Channel
+    ``command`` | [2:0] | The command field
+    
+    @subsection S-cmd-desc Command descriptions
+    
+    command[3:0] | Name     | Effect
```

Comment:
OK, so again, this is due to specifying the command to more than one channel, so in fact the table looks correct. However, the `command[3:0]` below it is too long (see my next comments).